articles/multifactor-authentication/troubleshooting.md

```diff
@@ -55,13 +55,9 @@ If you attempt to send more than ten SMS messages to your device within one hour
 
 When you exceed your messaging limit, you'll need to wait at least an hour after your request for your first message before requesting another. You will receive an additional attempt after the passage of each additional hour.
 
-## OTP-related issues
+## Rejected Codes
```

Comment:
just fyi - we didn't update this part (mfa codes)

Reply:
Yeah, the only reason I touched this doc is that I originally thought it applied to all, and then I stumbled on the doubled paragraph and such here and figured I might as well correct it.